kola/tests/misc/omaha.go

```diff
@@ -76,7 +75,6 @@ func OmahaPing(c cluster.TestCluster) error {
 
 	select {
 	case <-tc:
-		platform.Manhole(m)
```

Comment:
I recall when I first asked @crawford about using `platform.Manhole`, he told me to look here for an example usage, so I'm not sure if that is intentional or not.  It seems fine to me to remove it, though.